vitest.config.ts

```diff
@@ -3,5 +3,20 @@ import { defineConfig } from "vitest/config";
 export default defineConfig({
   test: {
     include: ["**/*.test.ts"], // Kun filer som slutter på .test.ts
+    exclude: ["**/node_modules/**", "**/.react-router/**"],
+    coverage: {
+      include: ["**/*.{ts,tsx}"],
+      exclude: [
+        "**/.react-router/**",
+        "**/routes.ts",
+        "**/*.config.ts",
+        "**/*.types.ts",
+        "**/mocks/**",
+        "**/sanity/**",
+        "app/entry.client.tsx",
+        "app/entry.server.tsx",
+      ],
+      provider: 'v8', // or 'istanbul'
```

Comment:
Trenger vi denne kommentaren? For meg som ikke er kjent med dette, bilr det litt forvirrende. Først trodde jeg `istanbul` var det samme som `v8`, men egentlig er det to ulike valg? Hvorfor velger man en fremfor den andre? 🤔